libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md

```diff
@@ -0,0 +1,76 @@
+# Sierra-F1 Autopilot by Sierra Aerospace
+https://www.sierraaerospace.in/
+
+## Features
+# Main Board:
+ - STM32H743 microcontroller
+ - 32MB QSPI flash
+ - BMI088+ICM42688 IMUs
+ - DPS310 barometer on SPI 
+ - SDMMC support
+ - 4 UARTs- 2 with HW flow control
+ - 6 PWM outputs with DSHOT capability
+ - Dual FDCAN
+ - RCIN port
+ - Dual power input with 2xI2C
+ - No co-processor
+
+# Daughter board:
+ -Ethernet PHY
+ -ICM42688/ICM45686/ADIS16507
+ -DPS310
+ -More PWMs with ALT function
+
+## Physical spec:
+ -Weight: 15g, Dimension: 35x45mm
+
+## UART Mapping
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART2 (Telem1, DMA-enabled)
+ - SERIAL2 -> UART8 (Telem2, DMA-enabled)
+ - SERIAL3 -> UART7 (GPS, NODMA)
+ - SERIAL4 -> UART6 (UART or RCIN, DMA-enabled)
+ - SERIAL7 -> USB2 (virtual port on same connector)
+
+The Telem1 & Telem2 port has RTS/CTS pins, the other UARTs do not have RTS/CTS.
+
+## Connectors
```

Comment:
needs diagram showing which jst-GH connector has what port....even if the ports internal pinout is standardized (I assume this is the standard? https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf)

Reply:
Added

Reply:
thanks...but the GPS+RCIN port does not appear to be compatible with that std I linked....so what is its pinnout...or do I link the wrong standard?

Reply:
I'll add pinouts in some time.